server/channels/api4/channel_bookmark_test.go

```diff
@@ -533,88 +586,6 @@ func TestEditChannelBookmark(t *testing.T) {
 		require.Equal(t, th.BasicUser2.Id, ucb.Updated.OwnerId)
 	})
 
-	t.Run("a user should be able to edit another usrer's bookmark", func(t *testing.T) {
```

Comment:
I fixed a typo in one of the first rebases and this test (mentioning `usrer's`) got duplicated... twice. Removed here